metal/resource_metal_project.go

```diff
@@ -56,7 +56,10 @@ func resourceMetalProject() *schema.Resource {
 				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
 					return strings.EqualFold(strings.Trim(old, `"`), strings.Trim(new, `"`))
 				},
-				ValidateFunc: validation.StringMatch(uuidRE, "must be a valid UUID"),
+				ValidateFunc: validation.Any(
+					validation.StringMatch(uuidRE, "must be a valid UUID"),
+					validation.StringIsEmpty,
+				),
```

Comment:
Looks good. I can't say if it fixes the problem though, as haven't been able to invoke/replicate the error from 
#201